app/controllers/internal/configs_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
 
   def config_params
     allowed_params = %i[
-      default_site_email mascot_user_id
```

Comment:
removed default_site_email and used a hash of email_addresses for the different types